tensorflow/compiler/xla/tests/filecheck.cc

```diff
@@ -40,7 +40,7 @@ StatusOr<bool> RunFileCheck(const std::string& input,
 
   // Invoke FileCheck to check whether input matches `pattern`.
   const char* file_check_path_suffix =
-      "org_tensorflow/external/llvm/FileCheck";
+      "org_tensorflow/external/llvm-project/llvm/FileCheck";
```

Comment:
Why this change?

Reply:
It is required as a consequence of this commit : https://github.com/tensorflow/tensorflow/commit/d6ba353dd974f3d883734e96653cdee7fe6abfbc

The path of the`FileCheck` executable changed after that commit, and that change is reflected here.

Reply:
Right, but why the tests are passing now without this change?

Reply:
They are not passing on our end without this change. I do not have visibility into your end

Reply:
> I do not have visibility into your end

You are right that tests are run in a different environment inside Google. However, presubmits which are being run by Github bot externally should be run inside the environment identical to yours, and you sort of do have visibility into it, as the bot should log all the actions it performs and should start from a reproducible state.

In any case, if the tests don't pass for you without this change, could you file a separate PR with this change only, and we could discuss/check it there?

Reply:
filed PR https://github.com/tensorflow/tensorflow/pull/35751 as per your reqeust